packages/infrastructure/kube_external_dns/main.tf

```diff
@@ -29,13 +29,47 @@ locals {
   namespace = module.namespace.namespace
 
   all_roles = toset([for domain, config in var.route53_zones : config.record_manager_role_arn])
-  config = { for role in local.all_roles : role => {
-    labels           = { role : sha1(role) }
-    zone_ids         = [for domain, config in var.route53_zones : config.zone_id if config.record_manager_role_arn == role]
+
+  cloudflare_config = { for domain, config in var.cloudflare_zones : domain => {
+    labels   = { domain : sha1(domain) }
+    provider = "cloudflare"
+
+    included_domains = [domain]
+    excluded_domains = [
+      for excluded_domain in var.cloudflare_zones :
```

Comment:
The set of possible excluded domains include _all_ domains (not just cloudflare domains). The same applies to route53 domains.

Reply:
@fullykubed updated